indexer/migrations/20230523_create_schema.sql

```diff
@@ -170,8 +208,8 @@ CREATE TABLE IF NOT EXISTS l1_bridge_deposits (
     -- Deposit information
 	from_address     VARCHAR NOT NULL,
 	to_address       VARCHAR NOT NULL,
-	l1_token_address VARCHAR NOT NULL,
-	l2_token_address VARCHAR NOT NULL,
+    l1_token_address VARCHAR NOT NULL, -- REFERENCES l1_tokens(token_address), uncomment me in future pr
```

Comment:
Also we dont want `l1/l2_token_address` in this table. `transaction_deposits/transaction_withdrawals` are for the L2ToL1MP and OptimismPortal interactions.

These fields belong in `l1_bridge_deposit` and `l2_bridge_withdrawal` which is the highest layer (standard bridge) and refers to tokens

Reply:
ah makes sense calling that a blocker since this pr is the one that comes before adding tokens to api (something I'm holding off on doing til processor is changed)

Reply:
wait looks like it's in the right table actually?